src/base/file/file_test.cc

```diff
@@ -5,21 +5,24 @@
 
 #include <third_party/gtest/exported/include/gtest/gtest.h>
 
-#include <fcntl.h>
+#include STL(fstream)
+#include STL_EXPERIMENTAL(filesystem)
+#include STL(system_error)
 
 namespace dist_clang {
 namespace base {
 
 TEST(FileTest, Read) {
   const auto expected_content = "All your base are belong to us"_l;
   const base::TemporaryDir temp_dir;
-  const String file_path = String(temp_dir) + "/file";
+  const Path temp_dir_path = temp_dir.GetPath();
```

Comment:
As `tmp_dir.GetPath()` looks more explicit than `operator Path()`.
But that's matter of taste, so let get the operator way...